harrisonDir/harrisonHomework.txt

```diff
@@ -0,0 +1,113 @@
+PART 1:
+
+a) true, because == treats strings and numbers as being equal
+
+b) false, because === does NOT treat strings and numbers as being equal (i.e. === cares about type)
+
+c) true, for the same reason as a)
+
+d) false, because while x == "x" is true, x == "x " is false
+
+e) false, because the strings are different
+
+f) true, because the operator = outputs the input on the right and changes x to the value of the input on the right
+
+g) true, because declaring var x does not define it, and it remains undefined
+
+h) NaN, because < takes two numbers as its inputs and will not know how to evaluate two string inputs
+
```

Comment:
Javascript will actually give these two strings value based on their first alpha-character, and then move on through the rest of the characters until it finds a number higher than the rest. In this case '9' > '1', but similarly '925' > '9074' because while '9' == '9', '2' > '0', which causes the comparison to finish, returning (in this case) true.